### DISCUSSION TITLE
Add offset to «in X days» for full day events

### DISCUSSION THREAD

**Discussion by jmgiaever** [totaldebug/atomic-calendar-revive#930] [Support]
Hi,

Is it possible to add an offset (+1) to full day events. It's confusing that it uses this term:

Today is the 27th
Upcoming event is the 30th
ACR says: «in two days»

so on the 29th it will say «in 0 days». Doesn't make sense. Say on the 27th: «in 3 days» and on the 29th «in 1 day».

**Comment by marksie1988** [comment]
Hi the relative time until an event is calculated by DayJS the library used for date formatting. 

I could potentially see if a custom calculation could be added but I'm not sure when.

**Comment by jmgiaever** [comment]
That would be great. It's a bit confusing as it's now. I mostly use the calendar for birthdays, trash collection etc, and I've already been early with the birthday wishes many times enough now to ask.

Should I register it as a feature request?
